divvy/cloudfilebrowser.js

```diff
@@ -95,13 +103,10 @@ var CloudElements = (function() {
             var docservicesnames = [];
             var docservicesimages = [];
 
-            //cedocumentconfig = data;
-
-            for(var x in data)
-            {
+            for (var x in data) {
                 var elementKey = data[x].key;
-                if(cedocumentconfig[elementKey] != null)
-                {
+                if (cedocumentconfig[elementKey] != null) {
+                    CloudElements.validateToken(elementKey)
```

Comment:
The `validateToken` is an asynchronous call, does that affect how the plugin works or it correctly disable invalid tokens?

Reply:
When testing the branch, the plugin continued to work correctly and the invalid token was disabled.

Reply:
Great, looks good to merge then!